### DISCUSSION TITLE
[Feature] Album that contains albums

### DISCUSSION THREAD

**Discussion by alberto8051** [immich-app/immich#1993] [Feature Request]
### The feature

Hi everyone, 

I think it could be useful than an Album can contains another album, so there's no need to create duplicates albums. Sorry for my bad english. I'm from Spain.  I'll try to explain this.
Suposse you have a collection of your holidays, for instance, in Spain ;-). You visited Madrid, Segovia and Toledo, so you create an album for each city. But if you want create an album fro Spain, you need to select again the photos for Madrid, Segovia and Toledo. 

Thank You.
Best Regards.
Alberto 

### Platform

- [ ] Server
- [x] Web
- [ ] Mobile

**Comment by vedpatel765** [comment]
I think what you’re trying to explain (and I’m completely on board if it happens!) is album folders, which is what the stock apple photos has. You can group a bunch of albums into one folder. Hope this helps :)

**Reply by alberto8051** [reply]
Thank you for your reply. I don't use Apple photos. I have an Android Phone and a digital camera, so I import the mobile photos using the app and the camera photos using the web and immich-cli, so I make the albums using the web. 

B. Regards
Alberto

**Comment by bo0tzz** [comment]
I think we're unlikely to implement this, but we do want to add support for tagging assets in the future. That would allow you to achieve pretty much the same thing.

**Reply by alberto8051** [reply]
Thank you for your reply and for your work. This was just a suggestion for future features.  
Immich is a great program. 

Best regards.
Alberto

**Comment by linus-ha** [comment]
See  #1010
